ext/pycall/pycall.c

```diff
@@ -921,6 +922,8 @@ pycall_extract_kwargs_from_ruby_hash(VALUE key, VALUE value, VALUE arg)
   if (Py_API(PyDict_SetItemString)(kwargs, key_cstr, pyvalue) < 0) {
     return ST_STOP;
   }
+  pycall_Py_DecRef(pyvalue);
+
   return ST_CONTINUE;
```

Comment:
I think the added `DecRef` should also be inserted before `return ST_STOP`.
How about rewriting the above piece of code by this?

```suggestion
  int res = Py_API(PyDict_SetItemString)(kwargs, key_cstr, pyvalue);
  pycall_Py_DecRef(pyvalue);
  return (res < 0) ? ST_STOP : ST_CONTINUE;
```

Reply:
OK